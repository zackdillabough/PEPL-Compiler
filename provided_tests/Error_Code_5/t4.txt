POLY F1 = x + 1;
POLY F4 = x^2 + x + 4;
POLY F3(X) = X - 1;
START
INPUT X;
F4(Y);
F1(Z);
F3(F1(0));
F4(F3(F4(F3(F4(F3(F4(F3(F4(F3(F4(F3(F4(F3
(F4(F3(F4(F3(F4(F3(P))))))))))))))))))));
363 2 2 1 0
