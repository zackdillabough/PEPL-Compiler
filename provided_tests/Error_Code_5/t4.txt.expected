Error Code 5: 6 7 10 